system/Database/BaseBuilder.php

```diff
@@ -2136,6 +2150,28 @@ public function setQueryAsData($query, ?string $alias = null, $columns = null):
         return $this;
     }
 
+    /**
+     * Gets $QBOptions for use in Model
+     */
+    public function getQBOptions(): array
```

Comment:
This method destroys encapsulation. Is there any way to remove it?

Reply:
I would have to implement all the QBOptions methods is Model. Otherwise I need some way of accessing and destroying these values from outside of Builder.